File: 09610.py
q1 = 0; q2 = 0; q3 = 0; q4 = 0; axis = 0
for i in range(int(input())):
    x, y = map(int, input().split())

    if not x or not y: axis += 1
    elif x > 0 and y > 0: q1 += 1
    elif x < 0 and y > 0: q2 += 1
    elif x < 0 and y < 0: q3 += 1
    else: q4 += 1

print("Q1: %d" % q1)
print("Q2: %d" % q2)
print("Q3: %d" % q3)
print("Q4: %d" % q4)
print("AXIS: %d" %axis)

File: 09610.cpp
#include <iostream>
using namespace std;

int main() {
	int n; cin >> n;

	int q1 = 0, q2 = 0, q3 = 0, q4 = 0, axis = 0;
	for (int i = 0; i < n; i++) {
		int x, y; cin >> x >> y;

		if (!x || !y) axis++;
		else if (x > 0 && y > 0) q1++;
		else if (x < 0 && y>0) q2++;
		else if (x < 0 && y < 0) q3++;
		else q4++;
	}

	cout << "Q1: " << q1 << endl;
	cout << "Q2: " << q2 << endl;
	cout << "Q3: " << q3 << endl;
	cout << "Q4: " << q4 << endl;
	cout << "AXIS: " << axis << endl;
}
